Source/WebCore/css/htmlSwitchControl.css

```diff
@@ -25,5 +25,10 @@
 @namespace "http://www.w3.org/1999/xhtml";
 
 input[type="checkbox"][switch] {
+    appearance: auto;
```

Comment:
This might not be needed as it would already get this from `input[type="checkbox"]` I think.